src/__tests__/footer.test.js

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable no-undef */
+import { render, screen, cleanup } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import Footer from "../components/Footer";
+import React from "react";
+import "@testing-library/jest-dom";
+import { configureStore } from "@reduxjs/toolkit";
+import rootReducer from "../reducers/index";
+import { Provider } from "react-redux";
+
+window.scrollTo = jest.fn();
+
+afterEach(() => {
+  cleanup();
+  jest.resetAllMocks();
+});
+afterAll(() => {
+  jest.clearAllMocks();
+});
+
+const store = configureStore({ reducer: rootReducer });
+
+afterEach(() => {
```

Comment:
Redundant function call needs to cleaned up.

Reply:
Fixed